scripts/tests/perf/test.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -xe
```

Comment:
I usually do `set -euo pipefail` but I can't remember what it does. Is one way better? Can that go in the scripts README over in #3679?

Reply:
Yeah, good call, I'll update it here and that PR

Reply:
Thanks. Found a reference here btw https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425